### ISSUE TITLE
Blank screen on Startup? Linux Mint 17.2

### ISSUE THREAD

**Issue by groovenectar** [raelgc/scudcloud#219]
When I start Scudcloud I have been getting a blank screen (with menu/titlebar) on startup, it usually lasts for a while with no response/loading/error or indication of what's happening... It had been loading fairly quickly before.

Did some poking around the issues on GitHub, and saw the debug option, so I ran:

`scudcloud --debug=True`

When it started, the blank screen happened again, and for some time. Tried right-clicking and "Inspect" and it immediately started loading.

I exited, tried launching with debug again. Blank screen. Right-clicked, and it immediately started loading before even inspecting.

Exited, tried without the debug option. Blank screen for a little while, BUT it did start loading unlike before.

Now, it seems to be working almost normally even through the launcher, with just a few seconds before it starts loading the page.

I had thought it was happening because I deleted all the trusted certificates on the machine (OS level, not browsers, and I figure that ScudCloud is using some kind of browser "WebView"?). Websites do load in all browsers on the machine (including *.slack.com) even if not super fast. I added back GeoTrust certificates because it appears to be what slack.com is using. But that didn't seem to help at all. Only after finding and launching with --debug did things FINALLY start getting back to normal, but... slowly.... back to aaalmost normal.....

So, this issue is probably really strange, and I wish I could be more clear about it. Could it be my Internet connection? Any suggestions on how to resolve / report more useful info if it happens again?

**Comment by raelgc** [comment]
Usually this is just related to slow connection. You're right, the program is based in `QWebView` library, which is full HTML engine.

**Comment by raelgc** [comment]
Duplicate of #158

**Comment by groovenectar** [comment]
I think that's a feasible explanation.. I wish I could debug it more effectively, but net speed could easily be the cause, thanks...
